modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+
+    'projection_db_server'  => $self->o('databases_server'),
+    'projection_db_port'    => $self->o('databases_port'),
+
+    databases_to_delete => ['projection_db',],
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps  => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+
+    'primary_assembly_dir_name'              => 'Primary_Assembly',
+    'refseq_cdna_calculate_coverage_and_pid' => '0',
+    'contigs_source'                         => 'ena',
+
+    'min_toplevel_slice_length'   => 250,
+
+    ensembl_analysis_script           => catdir($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts'),
+    flag_potential_pseudogenes_script => catfile($self->o('ensembl_analysis_script'), 'genebuild', 'flag_potential_pseudogenes.pl'),
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+
+    'cesar_path' => catdir($self->o('software_base_path'),'opt','cesar','bin'),
+
+# Max internal stops for projected transcripts
+    'projection_pid'                        => '50',
+    'projection_cov'                        => '50',
+    'projection_max_internal_stops'         => '1',
+    'projection_calculate_coverage_and_pid' => '1',
+
+########################
+# db info
+########################
+
+    'projection_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_proj_'.$self->o('release_number'),
+      -host   => $self->o('projection_db_server'),
+      -port   => $self->o('projection_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'projection_source_db' => {
+      -dbname => $self->o('projection_source_db_name'),
+      -host   => $self->o('projection_source_db_server'),
+      -port   => $self->o('projection_source_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'projection_lastz_db' => {
+      -dbname => $self->o('projection_lastz_db_name'),
+      -host   => $self->o('projection_lastz_db_server'),
+      -port   => $self->o('projection_lastz_db_port'),
+      -user   => $self->o('user_r'),
+      -pass   => $self->o('password_r'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'selected_projection_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_sel_proj_'.$self->o('release_number'),
+      -host   => $self->o('projection_db_server'),
+      -port   => $self->o('projection_db_port'),
```

Comment:
Could you please use selected_projection_db_* instead of projection_db_*?

Reply:
Updated